pkg/apis/kops/cluster.go

```diff
@@ -389,6 +389,8 @@ type LoadBalancerAccessSpec struct {
 	UseForInternalApi bool `json:"useForInternalApi,omitempty"`
 	// SSLCertificate allows you to specify the ACM cert to be used the LB
 	SSLCertificate string `json:"sslCertificate,omitempty"`
+	// SSLPolicy allows you to overwrite ELB listener's Security Policy
```

Comment:
along with the same change in `v1alpha2/cluster.go` and `make crds`:
```suggestion
	// SSLPolicy allows you to overwrite the LB listener's Security Policy
```

Reply:
Updated it. Thank you.